### PR TITLE
Add batch support for softmax operation

### PR DESCRIPTION
**Pull request by lisa0314**
Fixes [intel/webml-polyfill#152](https://github.com/intel/webml-polyfill/issues/152)
@fujunwei @huningxin  Please help to review.

### CODE REVIEW

File: services/ml/compilation_impl_mac.mm [hunk 1]
```diff
@@ -880,30 +888,35 @@ void ComputeBNNSOffsetForImplicitPadding(
...
+      DLOG(ERROR) << "  beta " << beta << " is not supported.";
+      return false;
+    }
     operation.offset_x = 0;
     operation.offset_y = 0;
 
     if (@available(macOS 10.13, *)) {
       BNNSVectorDescriptor in_desc, out_desc;
+      int batch = input.dimensions[0];
+      LOG(ERROR) << "batch_size: " << batch;
```
Comment (fujunwei) [inline services/ml/compilation_impl_mac.mm]: 1, It's better to use `DLOG(INFO) << "batch_size: " << input.dimensions[0];`
2, Why does it only log index of 0?

File: services/ml/execution_impl_mac.mm [hunk 2]
```diff
@@ -348,16 +349,22 @@ bool GetMPSImageInfo(const OperandMac& operand,
...
+                int in_stride = 1, out_stride = 1;
+                for (size_t i = 1; i < operation_input.dimensions.size(); i++) {
+                  in_stride = in_stride * operation_input.dimensions[i];
+                }
+                for (size_t i = 1; i < operation_output.dimensions.size();
+                     i++) {
+                  out_stride = out_stride * operation_output.dimensions[i];
+                }
+                result =
+                    BNNSFilterApplyBatch(operation.filter, input_batch_size,
```
[the remark this answers is not in the document]
Comment (lisa0314) [reply services/ml/execution_impl_mac.mm]: This is the style after running 'git cl format'

**Comment by huningxin** [comment]
Thanks @lisa0314 ! Looks good. @fujunwei for review.

**Comment by fujunwei** [comment]
LGTM.

**Pull request merged**